platforms/ide/ide-plugins/src/main/java/org/gradle/plugins/ide/internal/tooling/ToolingModelServices.java

```diff
@@ -68,6 +70,7 @@ public void execute(ToolingModelBuilderRegistry registry) {
                     registry.register(new EclipseModelBuilder(gradleProjectBuilder, projectStateRegistry));
                     registry.register(ideaModelBuilder);
                     registry.register(gradleProjectBuilder);
+                    registry.register(new ResilientGradleBuildBuilder(buildStateRegistry, failedIncludedBuildsRegistry));
```

Comment:
Shouldn't we register such resiliency related things only when the "resilient flag" is set? In this case by the TAPI connection? Isn't that the reason we wanted to have such a flag in the first place?

Reply:
Since we decided that we Use a different interface to request the resilient model, there is currently no need to use the option.